### PR TITLE
Create the SourceCred.io Team Directory

### PR DESCRIPTION
**Pull request by Bex2594**
The 'Meet the Team' page on the SourceCred.io page website.

### CODE REVIEW

File: docs/resources/team.md [hunk 1]
```diff
@@ -4,3 +4,49 @@ description: Meet the SourceCred team!
...
+# LB 
+## (they/them)
+
+### SourceCred Branch/Job Description: Community Cultivation/Social Architect
+
+### *I'm a genderqueer artist and communicator. I got to be around for the conception of SourceCred, and later became a part of the team using my soft skills around interpersonal relationships and community structure. Now, I champion the Community Cultivation branch and work with our community to develop a wholehearted space that can scale with its values.*
+
+### Contact: LBS on Discord 
+
+-photo-
```
Comment (Bex2594) [inline docs/resources/team.md:29]: 6D92DE70-2D9E-492A-98A3-44E22C181259

File: docs/resources/team.md [hunk 2]
```diff
@@ -4,3 +4,49 @@ description: Meet the SourceCred team!
...
+# Bex
+## (she/her)
+
+### SourceCred Branch/Job Description: Community Cultivation - Building community through accessibility and support
+
+### *Traveller, skackliner, boulderer and musician. A bit frisky. I'd love to see a world where SourceCred is widely used, because people would have the freedom to work on what brings them joy.*
+
+### Contact: Bex on Discord 
+
+-photo-
```
Comment (Bex2594) [inline docs/resources/team.md:51]: 458FDF16-0F34-4BF3-8AA8-D905F649C522

File: docs/resources/team.md [hunk 3]
```diff
@@ -4,3 +4,49 @@ description: Meet the SourceCred team!
...
+### *I'm a genderqueer hacker, a joyful hacker, and a bad philosopher. I've long been fascinated by the intersection of economics, philosophy, and technology, and SourceCred is an attempt to synthesize those perspectives into a tool to build new, more equitable ways of working together. I also love snowboarding and partner dancing.*
+
+### Contact: decentralion, across platforms (GitHub, Discord, Twitter, etc)
+
+-photo-
+
+# LB 
+## (they/them)
+
+### SourceCred Branch/Job Description: Community Cultivation/Social Architect
```
Comment (Bex2594) [inline docs/resources/team.md:23]: Community Cultivation Lead & Documentation Lead

File: docs/resources/team.md [hunk 4]
```diff
@@ -4,3 +4,49 @@ description: Meet the SourceCred team!
...
+# Seth
+## (he/him)
+
+### SourceCred Branch/Job Description: Creator of writings, hacky code, art, and thots on decentralized governance. Community builder, optimist, unlicensed therapist and cryptonaut.
+
+### *Here for the decentralized revolution, building a better alternative, and creating community with the amazing, talented people in SourceCred. When not working meditating, surfing, creating art and making conversation.*
+
+### Contact: s_ben on Discord, @zen_bacon (Twitter)
+
+-photo-
```
Comment (Ravipp) [inline docs/resources/team.md:40]: ![image of ben](https://github.com/sourcecred/docs/blob/master/images/team_photos/seth.png0

**Comment by decentralion** [comment]
I'm closing this because it's old and has conflicts. Not sure if this is in beta website scope, if it is we can make a new PR at that time.

**Pull request closed without merging**